.github/workflows/stable-spec-tests.yml

```diff
@@ -8,7 +8,7 @@ on:
   workflow_dispatch:
 
 env:
-  FIXTURES_TAG: "verkle@v0.0.7-alpha-4"
+  FIXTURES_TAG: "verkle@v0.0.7-alpha-5"
```

Comment:
This new alpha is baking. It will have fixed the `parentStateRoot` field renaming.